src/toxcore/tcp/client/connections.rs

```diff
@@ -279,6 +280,25 @@ impl Connections {
         }
     }
 
+    /// Get up to `count` random TCP relays we are connected to.
+    pub fn get_random_relays(&self, count: u8) -> Vec<PackedNode> {
+        let relays = self.clients
+            .read()
+            .values()
+            .filter(|client| client.is_connected())
+            .map(|client| PackedNode::new(client.addr, &client.pk))
+            .collect::<Vec<_>>();
+
+        if relays.is_empty() {
+            return Vec::new();
+        }
+
+        // TODO: shuffle relays instead
```

Comment:
Why todo?

Reply:
Because with libsodium we don't have a function for random shuffle. It can be done later with `rand` crate.